src/module-analyzer.js

```diff
@@ -214,7 +214,7 @@ export class ModuleAnalyzer {
     let vs;
     let resourceModule;
 
-    resourceModule = this.cache[moduleId];
+    resourceModule = this.cache[moduleId] && this.cache.hasOwnProperty(moduleId);
     if (resourceModule) {
       return resourceModule;
     }
```

Comment:
this looks strange to me- would that return `true` instead of what's in the cache?

Should it be:
```js
if (this.cache.hasOwnProperty(moduleId)) {
  return this.cache[moduleId];
}
```